_layouts/page.html

```diff
@@ -86,7 +86,7 @@ <h3><a href="/about">About</a></h3>
           <ul>
             <li><a href="/about#history">History</a>
             <li><a href="/about#sponsors">Sponsors</a>
-            <li><a href="https://groups.google.com/g/sandstorm-dev">Contact</a>
+            <li><a href="https://groups.io/g/sandstorm-user-group">Contact</a>
```

Comment:
+1 that group landing pages link to each other

Reply:
I slid that change into those landing pages only a few minutes ago, so I am glad you approve. :D